### PR TITLE
Parses revenue value and logger integration

### PR DESCRIPTION
**Pull request by rashidsp**

### CODE REVIEW

**Comment by rashidsp** [comment]
@aliabbasrizvi To accept 4.0 and '4.0' as valid input rather than only int(4)
View https://github.com/optimizely/php-sdk/pull/91
And integrated logger to log validation messages.

File: optimizely/helpers/event_tag_utils.py [hunk 1]
```diff
@@ -19,34 +19,71 @@
...
     return None
 
   raw_value = event_tags[REVENUE_METRIC_TYPE]
 
-  if isinstance(raw_value, bool):
+  if not type(raw_value) in (float, int, str):
+    logger.log(enums.LogLevels.WARNING, 'Revenue value is not an integer or float or a string.')
     return None
 
-  if not isinstance(raw_value, numbers.Integral):
```
[the remark this answers is not in the document]
Comment (oakbani) [reply optimizely/helpers/event_tag_utils.py]: @mikeng13 Can you please give your input on this? We already have revenue value parsing implemented in PHP and Ruby. https://github.com/optimizely/php-sdk/pull/91. 
Please note that 4.1 is not accepted as a valid revenue value while 4.0, "4", "4.0" are valid. 

Javascript is parsing 13 from 13.37. https://github.com/optimizely/javascript-sdk/blob/master/packages/optimizely-sdk/lib/utils/event_tag_utils/index.tests.js#L46

[the remark this answers is not in the document]
Comment (nchilada) [reply optimizely/helpers/event_tag_utils.py]: @oakbani oh, interesting, I didn't know that other SDKs already do some kind of advanced parsing. Seems okay if we accept a consistent set of string formats and if we ignore non-integer values.

@mikeng13 @ceimaj can you all follow up here? *TL;DR* we need to figure out how to parse revenue across our SDKs

**Comment by aliabbasrizvi** [comment]
I am assuming this change is not being worked on. Going to close this. Please re-open if you feel otherwise.

**Pull request closed without merging**